.github/workflows/slow-checks.yml

```diff
@@ -28,15 +32,15 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@1.2.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.6.0
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Run Tests and Collect Coverage
-        run: poetry run -- nox -s test:integration -- --coverage
+        run: poetry run -- nox -s test:integration -- --coverage --db-version ${{ matrix.exasol-version }}
```

Comment:
Please verify with @tkilias if CLI option `--db-version` is really correct here or wheter it should be `--itde-db-version`, see another ticket I will send you a reference.